.circleci/config.yml

```diff
@@ -21,13 +21,10 @@ workflows:
       - setup_and_typescript_tests
       - e2e_new_structure_tests:
           requires:
-            - e2e_navigation_and_misc
-      - e2e_navigation_and_misc:
-          requires:
-            - setup_and_typescript_tests
+            - e2e_library_and_collections
```

Comment:
We need to wait for `e2e_library_and_collections` since this is where webpack is built and GAE is set up. If we don't wait for `e2e_library_and_collections`, I get an error (`app.yaml not found`)

Reply:
That's because we have added `--skip-build` for the `e2e_new_structure_tests`, removing that will build the app.yaml file.

I'm fine with the current structure as well, so this is optional! :)